Ships as Raw

```diff
@@ -0,0 +1,860 @@
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|18.7
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|52
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|41
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+//This ship has a 9 mm section between the rear and front conjuctions.//
+
+Name|Acclamator I-class Assault Ship
+Type|Acclamator
+Keywords|Clone Transport
+Faction|Republic
+Size|Medium
+Points|66
+Hull|7
+defenseTokens|Sal Rd Br
+Command|3
+Squad|3
+Eng|4
+Speed1|1
+Speed2|10
+Speed3|010
+Speed4|0
+FrontShields|4
+SideShields|2
+RearShields|2
+FrontArmament|RRRKKK
+SideArmament|RRK
+RearArmament|RBK
+AntiSquadron|KK
+Upgrade|Title Officers TurboLaser Offensive Offensive  Weapon Team Ordance
+FrontArcOffset|24.6
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|18.7
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|52
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|41
+//How far from the rear of the ship do the rear hull zone lines meet//
+//This ship has a 9 mm section between the rear and front conjuctions.//
+
+Name|Munificent-class Comms Frigate
+Type|Munificent
+Keywords|Droid Bombard Comms
+Faction|Separatist
+Size|Medium
+Points|70
+Hull|6
+defenseTokens|Sal Rd Br
+Command|2
+Squad|3
+Eng|4
+Speed1|1
+Speed2|12
+Speed3|0
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|2
+FrontArmament|RRBB
+SideArmament|RRR
+RearArmament|RR
+AntiSquadron|R
+Upgrade|Title Officers TurboLaser Offensive Support Team Ion Cannon
+FrontArcOffset| 7.9
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|3.2
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|53
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|49
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Munificent-class Star Frigate
+Type|Munificent
+Keywords|Droid Bombard Transport
+Faction|Separatist
+Keyword|None
+Size|Medium
+Points|73
+Hull|6
+defenseTokens|Sal Rd Br
+Command|2
+Squad|3
+Eng|4
+Speed1|1
+Speed2|12
+Speed3|0
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|2
+FrontArmament|RRBB
+SideArmament|RRR
+RearArmament|RR
+AntiSquadron|RK
+Upgrade|Title Officers TurboLaser Defensive Suport Team Ion Cannon
+FrontArcOffset| 7.9
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|3.2
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|53
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|49
+//How far from the rear of the ship do the rear hull zone lines meet//

```

Comment:
I'm not wild about the comments on the geometry lines being on each instance of the ship.

Definitely have them on the first one or 2.  

Not concerned enough to ask you to remove them though if you don't want to.  Just registering that I'm not to keen on them.

Reply:
This was in there to act as a guide if someone wanted to make there own ships.  So thought I would describe what they meant.